### DISCUSSION TITLE
Need help for Docker configuration

### DISCUSSION THREAD

**Discussion by geronimo-78** [john30/ebusd#1306] [Q&A]
I would like to control my gas boiler via an Ebusd interface which is 192.168.0.76.
On my Synology, I created a docker with the john.30-ebusd-1 image, but I have configuration problems. What should we put in EBUSD_Foreground?
Attached is a copy of the docker configuration.
![John30-ebusd-parameter1](https://github.com/user-attachments/assets/1cbd64dd-3908-4a0b-a102-fc4d75598b7b)
![John30-ebusd-parameter2](https://github.com/user-attachments/assets/d5c8637c-6b9e-4ffb-b15e-65f56fb0aba7)
Thanks for your help.
Sincerely
Jean DOSTERT

**Comment by Koky05** [comment]
Do not need to specify every variable.
I did only for:
```
environment:
      - TZ=Europe/Bratislava
      - EBUSD_DEVICE=ens:192.168.2.31:9999
      - EBUSD_ARCH=amd64
      - EBUSD_SCANCONFIG=full
      - EBUSD_CONFIGPATH=/etc/ebusd/ebusd-configuration/ebusd-2.1.x/en
      - EBUSD_CONFIGLANG=EN
      - EBUSD_LOGAREAS=all
      - EBUSD_LOGLEVEL=notice
      - EBUSD_LATENCY=200000
      - EBUSD_ACQUIRETIMEOUT=100000
      - EBUSD_RECEIVETIMEOUT=100000
      - EBUSD_ACQUIRERETRIES=10
      - EBUSD_SENDRETRIES=10
      - EBUSD_ACCESSLEVEL=*
      - EBUSD_MQTTHOST=192.168.2.7
      - EBUSD_MQTTPORT=1883
      - EBUSD_MQTTCLIENTID=ebusd
      - EBUSD_MQTTUSER=${MQTT_USER}
      - EBUSD_MQTTPASS=${MQTT_PASSWORD}
      - EBUSD_MQTTTOPIC=ebusd
      - EBUSD_MQTTJSON=
      - EBUSD_MQTTINT=/etc/ebusd/mqtt-hassio.cfg
      - EBUSD_MQTTVERBOSE=
      - EBUSD_MQTTRETAIN=
      - EBUSD_MQTTCHANGES=
      - EBUSD_MQTTVERSION=3.1.1
```

**Reply by geronimo-78** [reply]
Thank you for your reply
By directly putting your variables in Synology's Docker environment, the Ebusd program starts and stops immediately without any message.
So, I decided to build the container with the YAML template provided by John, but I came up against the variable EBUSD_CONFIGPATH = "/config", knowing that I have mounted the config directory containing the .cvs file of my boiler, but I have the following message: "[main error] error reading config files from “config”/: ERR: element not found, last error: "
A part of the configuration of the container :
![John30-ebusd-parameter3](https://github.com/user-attachments/assets/33b20e8b-a1e3-4997-963b-a0a86c9efafc)

Can you help me
Sincerely
Jean

**Reply by vjonas** [reply]
I think this is due to permissions issues on the /etc/ebusd/config folder and files.
Make sure to add       - EBUSD_CONFIGPATH=/etc/ebusd/config to the env variables as mentioned by @Koky05
